lib/websocket_lwt.ml

```diff
@@ -50,37 +74,16 @@ module Connected_client = struct
       oc;
       http_request;
       standard_frame_replies = false;
+      read_frame;
+      read_frame_std;
+      send_frame;
```

Comment:
Is it necessary to move these three functions on the record? They are never mutated, and even if it was necessary to change in the future we could still change it since the type is not visible.